src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooter.tsx

```diff
@@ -60,17 +66,31 @@ const SendTilBeslutterFooter: React.FC<{
 
     return (
         <>
-            <Footer>
-                {feilmelding && <AlertStripeFeilPreWrap>{feilmelding}</AlertStripeFeilPreWrap>}
-                <MidtstiltInnhold>
-                    <StyledHovedknapp
-                        onClick={sendTilBeslutter}
-                        disabled={laster || kanSendesTilBeslutter === false}
-                    >
-                        Send til beslutter
-                    </StyledHovedknapp>
-                </MidtstiltInnhold>
-            </Footer>
+            <DataViewer response={{ personopplysningerResponse }}>
+                {({ personopplysningerResponse }) => {
+                    const harVerge = personopplysningerResponse.vergemål.length !== 0;
+                    return harVerge ? (
```

Comment:
Hvordan ser dette ut for beslutter? Burde det være noe info som viser hvem som er mottakere istedenfor hele knappeforløpet?

Reply:
Fikses i annen PR.